tests/plugins/ml/ml/models.yaml

```diff
@@ -51,6 +51,37 @@ chapters:
         hits:
           hits:
             - _score: 1
+  - synopsis: Search model.
```

Comment:
This one should go into `models/search.yaml` where the model is created in a prologue.